src/IoC/IoC.java

```diff
@@ -0,0 +1,19 @@
+package IoC;
+
+import java.util.Map;
+
+public class IoC {
+    private static IoC instance;
+    private Map<String,Object> myMap;
```

Comment:
This map should be final

Reply:
Rename this map to something more useful

Reply:
solved